docs/multicluster_test_example.md

```diff
@@ -0,0 +1,109 @@
+# Multi-cluster test example
+
+The following example and notes are based on the main
+[PR#4875](https://github.com/red-hat-storage/ocs-ci/pull/4875) for this feature.
+
+## Example test
+
+Following example is simple test which do basically nothing, just shows how to
+switch between cluster configuration contexts and how to access configuration
+of particular cluster without switching context to it.
+
+```python
+import logging
+import os
+
+from ocs_ci.utility.utils import exec_cmd
+
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+def test_multicluster_dahorak():
+    # print number of clusters to the log
+    log.info(f"Number of clusters: {config.nclusters}")
+
+    # iterate over the clusters and perform some actions on each cluster
+    for i in range(config.nclusters):
+        # switch context to the i-th cluster
+        config.switch_ctx(i)
+        # print the cluster name of the selected cluster
+        log.info(f"CLUSTER_NAME: {config.ENV_DATA['cluster_name']}")
+        # run oc command against the selected cluster
+        log.info(exec_cmd("oc version").stdout)
+        # print kubeconfig path of the selected cluster
+        log.info(os.environ["KUBECONFIG"])
+
+    # access configuration of particular cluster without switching context to it
+    log.info(f"Cluster name of first cluster: {config.clusters[0].ENV_DATA['cluster_name']}")
+    log.info(f"Cluster name of second cluster: {config.clusters[1].ENV_DATA['cluster_name']}")
+```
+
+## Execution of the example test
+
+I saved the above example test into `tests/test_multicluster_dahorak.py` file.
+Additionally I have two clusters and related cluster dirs:
+
+* cluster: `dahorak-test1`, cluster patch: `../clusters/dahorak-test1/`
+* cluster: `dahorak-test2`, cluster patch: `../clusters/dahorak-test2/`
+
+(Each cluster dir contains only `auth/kubeconfig` directory/file.)
+
+Now I'll run the test via following command:
+
+```bash
+$ run-ci multicluster 2 tests/test_multicluster_dahorak.py \
+    --ocsci-conf ocsci_conf.yaml --ocsci-conf conf/ocsci/skip_ocs_deploy.yaml \
+    --cluster1 --cluster-path ../clusters/dahorak-test1/ --cluster-name dahorak-test1 --ocp-version 4.10 \
+    --cluster2 --cluster-path ../clusters/dahorak-test2/ --cluster-name dahorak-test2 --ocp-version 4.9 --ocsci-conf config_for_cluster2.yaml
+```
+* On the second line, I provided two configuration files common for both clusters (I provided the `skip_ocs_deploy.yaml` file, to avoid additional checks and log messages in the console output).
```

Comment:
A part of similar explanation is present in https://github.com/red-hat-storage/ocs-ci/blob/master/docs/usage.md. Just FYI.

Reply:
Good point, I was gathering the info from there, but forgot to mention it in the doc - added the link in commit 983968c.